src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -4342,48 +4041,36 @@ namespace HVACMultiSpeedHeatPump {
         // SUBROUTINE INFORMATION:
         //       AUTHOR:          Lixing Gu
         //       DATE WRITTEN:    June 2007
-        //       MODIFIED:        na
         //       RE-ENGINEERED    Revised to calculate MSHP heat recovery rate based on EIR Chiller heat recovery subroutine
         // PURPOSE OF THIS SUBROUTINE:
         //  Calculate the heat recovered from MSHP
 
-        // Using/Aliasing
-        auto &MSHPWasteHeat = state.dataHVACGlobal->MSHPWasteHeat;
-        using FluidProperties::GetSpecificHeatGlycol;
-        using PlantUtilities::SafeCopyPlantNode;
-
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:4351]:(style),[constVariable],Variable 'MSHPWasteHeat' can be declared as reference to const